draft-ietf-taps-interface.md

```diff
@@ -389,10 +389,13 @@ Listener -> ConnectionReceived<Connection>
 // Only receive complete messages in a Conn.Received handler
 Connection.Receive()
 
+messageContext := NewMessageContext()
```

Comment:
Is this in the right place? Should it not be in the event handler where you send the message?